Resources/Prototypes/_Harmony/Catalog/Fills/Crates/armory.yml

```
@@ -44,3 +44,27 @@
       - id: SpeedLoaderLightRifle
         orGroup: ammo
         amount: 42
+
+- type: entity
+  id: CrateArmorySMGHarmony
+  suffix: Drozd
+  parent: [ CrateWeaponSecure, BaseSecurityContraband ]
+  name: SMG crate
+  description: Contains two high-powered, automatic rifles with four mags. Requires Armory access to open.
+  components:
+  - type: StorageFill
+    contents:
+      - id: WeaponSubMachineGunDrozd
+        amount: 2
+
+- type: entity
+  id: CrateArmoryShotgunHarmony
+  suffix: Kammerer
+  parent: [ CrateWeaponSecure, BaseSecurityContraband ]
+  name: shotgun crate
+  description: For when the enemy absolutely needs to be replaced with lead. Contains two Kammerer Shotguns, and some standard shotgun shells. Requires Armory access to open.
```

The order is incorrect, it should be type > abstract > parent > id > categories > name > suffix > description > components
```suggestion
  parent: [ CrateWeaponSecure, BaseSecurityContraband ]
  id: CrateArmoryShotgunHarmony
  name: shotgun crate
  suffix: Kammerer, Secure
  description: For when the enemy absolutely needs to be replaced with lead. Contains two Kammerer Shotguns, and some standard shotgun shells. Requires Armory access to open.
```